zio-http/src/main/scala/zhttp/service/ServerStart.scala

```diff
@@ -0,0 +1,31 @@
+package zhttp.service
+
+sealed trait ServerStart { self =>
```

Comment:
This isn't a domain, its data. You can simply use a product type here. ie. just have a case class called `Start(port: Int)`. That should do the job for now. Later we can see, if there is something else that we would like to add.

Reply:
Got it. I thought of it as a sum type, which currently has only port but could also have other types.
Ref: https://en.m.wikipedia.org/wiki/Tagged_union